src/rsz/src/Resizer.cc

```diff
@@ -272,8 +272,8 @@ void Resizer::removeBuffers(sta::InstanceSeq insts, bool recordJournal)
         logger_->warn(
             RSZ,
             97,
-            "Instance {} cannot be removed because it is not a buffer",
-            " or is a feedthrough port buffer",
+            "Instance {} cannot be removed because it is not a buffer,"
+            "functions as a feedthrough port buffer, or is constrained",
```

Comment:
Needs a space between `,` and `functions`

Reply:
Good catch!   Fixed.